frontend/openchat-shared/src/domain/crypto.ts

```diff
@@ -68,7 +68,7 @@ export const dollarExchangeRates: Record<string, number> = {
     sonic: to2SigFigs(dollarToICP / 0.0274),
     sneed: to2SigFigs(dollarToICP / 22.25),
     exe: to2SigFigs(dollarToICP / 0.0152),
-    taggr: to2SigFigs(dollarToICP / 1.0197),
+    taggr: to2SigFigs(dollarToICP / 0.6565),
```

Comment:
This looks wrong... the price on icpcoints is 0.1851 ICP

Reply:
But I think you picked the TAGGR/USD price rather than the TAGGR/ICP price.
It doesn't look like it has ever been as high as the price you put here.

Reply:
🤦 - corrected